puma/tests/hlplots/test_results.py

```diff
@@ -61,7 +58,7 @@ def test_add_taggers_from_file(self):
             data = get_dummy_2_taggers()
```

Comment:
Hi @samvanstroud 

This looks very good!
I have one question. Do we need these lines here? Can we use the `get_mock_file()` function instead and remove this file writing and the `get_dummy_2_taggers` import?

Reply:
agree! fixed